src/components/CurriculumComponents/CurricVisualiserLayout/index.tsx

```diff
@@ -1,5 +1,10 @@
 import { OakFlex } from "@oaknational/oak-components";
 import styled from "styled-components";
+import { PortableTextBlock } from "@portabletext/types";
+
+import { CurricSEOAccordion } from "../CurricSEOAccordion/CurricSEOAccordion";
+
+import { SubjectPhasePickerData } from "@/components/SharedComponents/SubjectPhasePicker/SubjectPhasePicker";
```

Comment:
Minor: We should probably move this type into `./src/utils/curriculum/types.ts` also, if you feel the motivation in this PR go for it.